README.md

```diff
@@ -1,2 +1,20 @@
 # solc-bin
-Current and historical (emscripten) binaries for Solidity
+
+This repository contains current and historical builds of the [Solidity Compiler](https://github.com/ethereum/solidity/).
```

Comment:
The link obviously won't work until https://github.com/ethereum/solidity/pull/10352 is merged.